packages/bson-transpilers/README.md

```diff
@@ -3,7 +3,7 @@
 [![downloads][5]][6]
 
 Transpilers for building BSON documents in any language. Current support
-provided for `shell` `javascript` and `python` as inputs. `java`, `c#`, `node`, `shell` and `python` as
+provided for `shell` `javascript` and `python` as inputs. `java`, `c#`, `node`, `shell`, `python` and `ruby` as
```

Comment:
Should this say `go`?

Reply:
Ah yeah, we should definitely include `go`.  Let me add that to [the ](https://github.com/mongodb-js/compass/pull/2991) so that this PR can remain language agnostic.  I added ruby here based for [this request](https://github.com/mongodb-js/compass/pull/2964#issuecomment-1092782188).